okta/plugin.spec.yaml

```diff
@@ -131,6 +164,33 @@ connection:
     description: Okta key
     type: credential_secret_key
     required: true
+triggers:
+  users_added_removed_from_group:
+    title: Monitor User Groups
+    description: Monitors a list of groups for user membership changes
+    input:
+      group_ids:
+        title: Group ID's
+        description: A list of group ID's
+        type: "[]string"
+        required: true
+      interval:
+        title: Interval
+        description: The time in seconds inbetween checks for changes to the groups users
+        type: integer
+        required: true
+        default: 300
+    output:
+      users_added_from_groups:
+        title: Additions
+        description: Users added to a group since the last check
+        type: "[]trigger_type"
```

Comment:
Can you rename this type? If we ever add another trigger to the plugin then this will become confusing.